Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:alpine3.6
+
+RUN pip install requests python-slugify pytest
```

Comment:
Could this be changed to use [pipenv](https://github.com/pypa/pipenv)? This way we can leverage the [Pipfile.lock](https://github.com/froi/data-pr-downloader/blob/master/Pipfile.lock)